FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -345,17 +365,6 @@ get_options()
 		log_Msg "T1w Brain: ${p_t1w_brain}"
 	fi
 
```

Comment:
Would be helpful to maintain a comment here that the check for `{p_t2w_image}` has been moved to the `check_mpp_compliance` function.  That's the problem with the adopted approach.  We have now spun off some relevant code into a function that is easily missed...

Likewise, include a comment for any other scripts that had an existing check that was moved off to `check_mpp_compliance`

Reply:
Added comment. No other script is strucured in the way that would require such a comment.